tests/qa_features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,657 @@
+<?php
+
+use Behat\Behat\Context\ClosuredContextInterface;
+use Behat\Behat\Context\TranslatedContextInterface;
+use Behat\Behat\Context\BehatContext;
+use Behat\Behat\Exception\PendingException;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Gherkin\Node\ScenarioNode;
+
+/**
+ * Features context for Behat feature testing
+ */
+class FeatureContext extends BehatContext {
```

Comment:
Actually, I wrote it this way so that we can merge into the main feature context. What are you seeing that is different? I think I am missing a few functions you've added since ~Friday, but otherwise don't know why they have to be separate.